### PR TITLE
Remove wait from Codeception Travis env

### PR DESCRIPTION
**Pull request by connorshea**
This doesn't speed up the test suite like I had hoped - or at least not by any noticeable amount - but it should make the tests more consistent across environments.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,30 +5,9 @@ sudo: required
 matrix:
   fast_finish: true
   include:
-    - name: "PHP 5.5 / MySQL 5.6"
-      php: "5.5"
-      dist: trusty
-    - name: "PHP 5.6 / MySQL 5.6"
-      php: "5.6"
-      dist: trusty
-    - name: "PHP 7.0 / MySQL 5.7"
```
Comment (Dillon-Brown) [inline .travis.yml]: I wonder if we should actually just remove PHP 7.0. Is there really much benefit from testing 7.0 alongside 7.1?

I guess ideally I'd like to split things up so we maybe do a daily build that runs through absolutely everything while still having a fairly speedy process for PRs.

Comment (connorshea) [reply .travis.yml]: I think Travis has some form of scheduled jobs that'd allow this? I certainly wouldn't be opposed.

Can we remove 5.5 while we're at it ;)

**Comment by connorshea** [comment]
I think this can be merged now.

**Comment by Mac-Rae** [comment]
Assessed by Travis :+1:

**Pull request merged**